circle.yml

```diff
@@ -49,23 +50,72 @@ step-library:
         name: Prepare
         command: |
           git submodule sync
+          brew install carthage
+
+  - &prepare-iphone6s-plus-ios-11
+      run:
+        name: Prepare iPhone 6s Plus iOS 11
+        command: xcrun instruments -w "iPhone 6s Plus (11.4) [" || true
+
+  - &prepare-iphone6s-plus-ios-12
+      run:
+        name: Prepare iPhone 6s Plus iOS 12
+        command: xcrun instruments -w "iPhone 6s Plus (12.0) [" || true
+
+  - &reset-simulators
+      run:
+        name: Reset simulators
+        command: |
+          bundle install
+          SNAPSHOT_FORCE_DELETE=1 bundle exec fastlane snapshot reset_simulators
+
+  - &verify-missing-localizable-strings
+      run:
+        name: Verify missing localizable strings
+        command: |
+          ./scripts/convert_string_files.sh
+          git diff --exit-code -- */*/*.lproj
 
   - &install-dependencies
       run:
         name: Install Dependencies
         command: carthage bootstrap --platform ios --cache-builds --configuration Release --no-use-binaries
 
-  - &build-test-MapboxCoreNavigation
+  - &build-test-MapboxCoreNavigation-ios-11
+      run:
+        name: Build and Test MapboxCoreNavigation
+        command: |
+          xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=11.4,name=iPhone 6 Plus' -project MapboxNavigation.xcodeproj -scheme MapboxCoreNavigation clean build test | xcpretty
+
+  - &build-test-MapboxCoreNavigation-ios-12
       run:
-        name: Build and test MapboxCoreNavigation
+        name: Build and Test MapboxCoreNavigation
         command: |
-          xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,id=9F32C487-FAEC-4ABE-B00D-159EC007E50D' -project MapboxNavigation.xcodeproj -scheme MapboxCoreNavigation clean build test | xcpretty
+          xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=12.0,name=iPhone 6 Plus' -project MapboxNavigation.xcodeproj -scheme MapboxCoreNavigation clean build test | xcpretty
 
-  - &build-MapboxNavigation
+  - &build-test-MapboxNavigation-ios-11
       run:
         name: Build MapboxNavigation
         command: |
-          xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,id=9F32C487-FAEC-4ABE-B00D-159EC007E50D' -project MapboxNavigation.xcodeproj -scheme MapboxNavigation clean build | xcpretty
+          xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=11.4,name=iPhone 6 Plus' -project MapboxNavigation.xcodeproj -scheme MapboxNavigation clean build test | xcpretty
+
+  - &build-test-MapboxNavigation-ios-12
+      run:
+        name: Build MapboxNavigation
+        command: |
+          xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=12.0,name=iPhone 6 Plus' -project MapboxNavigation.xcodeproj -scheme MapboxNavigation clean build | xcpretty
+
+  - &build-Example-Obj-C
+      run:
+        name: Build and Test Example-Objective-C
```

Comment:
We're not testing this yet, just building it. Perhaps adjust the name accordingly?